set pluginsdir to /usr/lib/meshlab
 TODO: Put a short summary on the line above and replace this paragraph
 with a longer explanation of this change. Complete the meta-information
 with other relevant fields (see below for details). To make it easier, the
 information below has been extracted from the changelog. Adjust it or drop
 it.
 .
 meshlab (1.3.3) precise; urgency=low
 .
   * new snapshot release
Author: Stefan Schlager <zarquon42@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- meshlab-1.3.3.orig/meshlab/src/common/pluginmanager.cpp
+++ meshlab-1.3.3/meshlab/src/common/pluginmanager.cpp
@@ -200,7 +200,7 @@ QString PluginManager::getBaseDirPath()
 
 QString PluginManager::getDefaultPluginDirPath()
 {
-  QDir pluginsDir(getBaseDirPath());
+  QDir pluginsDir("/usr/lib/meshlab");
   if(!pluginsDir.exists("plugins"))
     //QMessageBox::warning(0,"Meshlab Initialization","Serious error. Unable to find the plugins directory.");
     qDebug("Meshlab Initialization: Serious error. Unable to find the plugins directory.");
